main.go

```diff
@@ -560,7 +671,7 @@ func main() {
 		}
 
 		stopUpdateIndex := make(chan struct{})
-		go s.updateAutoretrieveIndex(time.Duration(intervalMinutes)*time.Minute, stopUpdateIndex)
+		go s.updateAutoretrieveIndex(time.Duration(intervalMinutes)*time.Second, stopUpdateIndex)
```

Comment:
this could look like;

```

arMgr, err  := autoretrieve.NewAutoretrieveEngine() /
if err != nil {
	return err
}
defer arMgr.Shutdown()
go arMgr.Run(context.Background(), time.Duration(intervalMinutes)*time.Second, stopUpdateIndex)